sql-plugin/src/main/scala/org/apache/spark/sql/rapids/datetimeExpressions.scala

```diff
@@ -135,6 +135,8 @@ abstract class GpuTimeMath(
 
   override lazy val resolved: Boolean = childrenResolved && checkInputDataTypes().isSuccess
 
+  protected val microSecondsInOneDay: Long = TimeUnit.DAYS.toMicros(1)
```

Comment:
Curious, what do the changes in this file have to do with sorting of struct columns?

Reply:
nothing in particular, it's minor refactor that I noticed in the prior reviews and thought may not  be worth a dedicated PR.

Reply:
I personally prefer such things to be separate when they're not touching files that wouldn't normally be touched by the changes.  Otherwise where does it end?  You could end up with a PR that would normally only touch 3 files but ends up touching 20 because of other 'minor refactors'.  It blurs the focus of the patch and increases its risk footprint.  It also means the commit headline belies the changes being made, since a number of "oh, by the way, here's some unrelated stuff as well" changes come along silently for the ride.

Reply:
I am usually mindful of these concerns, however considered this minor enough and quite localized. If you insist, I'll remove it.

Reply:
I insist. :smile:  Touching unrelated files is where I draw the line during reviews.